rollup.config.js

```diff
@@ -16,7 +16,7 @@ export default {
       sourcemap: true
     }
   ],
-  external(id) { return !/^[./]/.test(id); },
+  external: Object.keys(pkg.dependencies),
```

Comment:
Does that work, i.e. does rollup also recognize sub-path imports of dependencies in this case?

I want to make sure that depending on `foo/bar` is not suddenly an issue.

A simpler fix could be:

```suggestion
external(id) { return !id.startsWith('.'); },
```

Reply:
`startsWith('.')` is already covered by the current regex. However, windows replaces the import path with an absolute path, e.g. `C:\path\to\lib\index.js`.

To include sub-paths of dependencies, this should work:

```suggestion
  external(id) {
    return Object.keys(pkg.dependencies)
      .includes(id.split('/')[0]);
  },
```

What do you think?

Reply:
Just realized that this does not work for namespaced dependencies, as it will check `@namespace/package` against the split `@namespace` 🙈 

It should check for something like
```
!!Object.keys(pkg.dependencies).find(dep => {
    return id.startsWith(dep)
})
```

I'll create a follow-up